### PR TITLE
Update go-license installation instructions

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

**Comment by jaxesn** [comment]
It may be worth calling this script this out? https://github.com/aws/eks-anywhere-build-tooling/blob/main/build/lib/install_go_versions.sh

I always use that to keep my golang versions the same on my local as in the builder-base, it also installs go-licenses.

File: docs/development/attribution-files.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ look out for.
 
 If trying to generate these files outside of CI and have them match what CI produces, you will need the following:
 
-1. `go get github.com/google/go-licenses`
+1. `go install github.com/google/go-licenses@latest`
```
Comment (TerryHowe) [inline docs/development/attribution-files.md:51]:
```suggestion
1. `build/lib/install_go_versions.sh` will install the right go versions and go-licenses for that go version
```

**Pull request merged**